README.md

```diff
@@ -301,6 +301,45 @@ import exampleGo from '!!raw-loader!./code-example.go'
 ```
 ````
 
+### Mermaid graphs
```

Comment:
```suggestion
### Diagrams
```

Reply:
Also, looking @ where you added this is the right place. Why is this in "import and reference content"? Looks more like a formatting thing to me.

Reply:
+ make sure to have the heading included in the auto-generated ToC

Reply:
No, the mermaid plugin interprets the text, renders it into an image.

https://docusaurus.io/docs/next/markdown-features/diagrams